modules/apps/27-interchain-accounts/host/keeper/keeper.go

```diff
@@ -276,7 +277,15 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 
 // newModuleQuerySafeAllowList returns a list of all query paths labeled with module_query_safe in the proto files.
 func newModuleQuerySafeAllowList() []string {
-	protoFiles, err := gogoproto.MergedRegistry()
+	fds, err := gogoproto.MergedGlobalFileDescriptors()
+	if err != nil {
+		panic(err)
+	}
+	// create the files using 'AllowUnresolvable' to avoid
+	// unnecessary panic
```

Comment:
```suggestion
	// create the files using 'AllowUnresolvable' to avoid
	// unnecessary panic: https://github.com/cosmos/ibc-go/issues/6435
```
I think it might be nice to link to your issue, otherwise I'm not sure the comment would make much sense on first read